conf/kansha.cfg

```diff
@@ -46,19 +46,22 @@ activated = off
 [[[twitter]]]
 activated = off
 
-[mail]
-activated = off
+[locale]
+major = en
+minor = US
+
+[services]
+[[mail_sender]]
+activated = on
 smtp_host = localhost
-smtp_port = 25
+smtp_port = 1025
```

Comment:
'smtp_host' -> 'host' / 'smtp_port' -> 'port' because we already are in the 'mail_sender' section